src/modules/ekf_att_pos_estimator/estimator_utilities.cpp

```diff
@@ -38,6 +38,7 @@
  */
 
 #include "estimator_utilities.h"
+#include <algorithm>
```

Comment:
I would like to avoid baking in C++11 bits. It compiles now on this platform, but some higher-end platforms actually only have C++03.